SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+If you have discovered a security vulnerability in this project, please report it
+privately. **Do not disclose it as a public issue.** This gives us time to work with you
+to fix the issue before public exposure, reducing the chance that the exploit will be
+used before a patch is released.
+
+Please submit the report as a [security bug on the Chromium tracker](https://bugs.chromium.org/p/chromium/issues/entry?template=Security%20Bug).
+
+Please provide the following information in your report:
+
+- A description of the vulnerability and its impact
+- How to reproduce the issue
+- Make it clear that it's an Emscripten bug.
+
+This project is maintained by a team of volunteers on a reasonable-effort basis. As
+such, please give us 90 days to work on a fix before public exposure.
```

Comment:
```suggestion
Please give us 90 days to work on a fix before public exposure.
```
The most frequent contributors are paid to work on emscripten so "volunteers" doesn't seem right.

What do most projects (non-volunteer-only) have in terms of the number of days here?

Reply:
90 days is quite standard, actually. Though there's usually an exception for vulnerabilities that are being exploited in the wild, in which case the deadline drops to 7 days. That's [Google](https://about.google/appsecurity/)'s policy, for example.

Reply:
Done.